### ISSUE TITLE
a11y: missing variables in label and hint strings

### ISSUE THREAD

**Issue by ricardojorgerm** [carrismetropolitana/app#9]
https://github.com/carrismetropolitana/app/blob/92ee96a57869dc45f8468e413146697d4e9ef0da/src/components/lines/waypoints/PathWaypointNextArrivals/index.tsx#L83
https://github.com/carrismetropolitana/app/blob/92ee96a57869dc45f8468e413146697d4e9ef0da/src/components/lines/waypoints/PathWaypointNextArrivals/index.tsx#L84
https://github.com/carrismetropolitana/app/blob/92ee96a57869dc45f8468e413146697d4e9ef0da/src/components/lines/waypoints/PathWaypointNextArrivals/index.tsx#L104
https://github.com/carrismetropolitana/app/blob/92ee96a57869dc45f8468e413146697d4e9ef0da/src/components/lines/waypoints/PathWaypointNextArrivals/index.tsx#L105

These strings should reference the following variables to be read out:
- time
- headsign
- hours
- minutes
